### PR TITLE
Add NormalizeJsonString helper

### PR DESCRIPTION
**Pull request by sylviamoss**
Adding function that will be used by the next `amazon-secretsmanager-secret` data source. Also add a missing part to the data source acc test structure.

### CODE REVIEW

File: json/helper.go [hunk 1]
```diff
@@ -38,3 +38,24 @@ func Unmarshal(data []byte, i interface{}) error {
...
+// the JSON parser to normalize it, returns either a parsing
+// error or normalized JSON string.
+func NormalizeJsonString(jsonString interface{}) (string, error) {
+	var j interface{}
+
+	if jsonString == nil || jsonString.(string) == "" {
+		return "", nil
+	}
+
+	s := jsonString.(string)
```
Comment (azr) [inline json/helper.go:52]: This could be simplified a bit:

```suggestion
	s, ok := jsonString.(type)
	if !ok || s == "" {
	  return "", nil
	}
```

**Review by azr — APPROVED** [review]
Nice one ! I think it could be a good idea to add a tiny test ( I don't think it's mandatory )

**Review by nywilken — APPROVED** [review]
LGTM

**Comment by sylviamoss** [comment]
No need for this PR anymore.

**Pull request closed without merging**